### DISCUSSION TITLE
Call a workload function from a pipeline?

### DISCUSSION THREAD

**Discussion by sschuetz2bit** [microsoft/Microsoft-Fabric-workload-development-sample#48] [Q&A]
Is there a way we can trigger a custom workload function from a pipline?

As a work around we are calling the api via python in a notebook, but this does not seem the perfect way.

**Comment by JosEilersTwyzer** [comment]
@sschuetz2bit yes you can, you do need to use oauth2 to call the api from a data pipeline (which is what you also probably do within the notebook), follow the following instructions : https://www.tackytech.blog/how-to-call-fabric-rest-apis-from-fabric-data-pipelines/

In your case you want to use the Jobs api part, for example like with the following example relative url: 
`/workspaces/@{pipeline().parameters.workspaceId}/items/@{pipeline().parameters.itemId}/jobs/instances?jobType=Org.Sample.SampleWorkloadItem.ScheduledJob`
